src/nuclio/projects/project/functions/version/version-code/function-event-pane/test-events-logs/test-events-logs.component.js

```diff
@@ -85,7 +86,7 @@
          * @returns {boolean}
          */
         function hasAdditionalParameters(log) {
-            return lodash.chain(log).omit(REQUIRED_PARAMETERS).size().value() > 1;
+            return lodash.size(getParameters(log)) > 1;
```

Comment:
I wonder, why `> 1`? 🤔
Shouldn't this be `> 0`? If it should, you could use `!lodash.isEmpty(getParameters(log))` instead.